entrypoint.sh

```diff
@@ -5,15 +5,25 @@ KOC_SERVER_FILE_PATH=/data/KnockoutCityServer
 if [[ -d "$KOC_SERVER_FILE_PATH" && "${KOC_FORCE_SERVER_DOWNLOAD:-false}" == false ]]; then
   echo "Server files already downloaded. Skipping download..."
 else
-  if [[ -z "$KOC_SERVER_DONWLOAD_URL" ]]; then
+  if [[ -z "$KOC_SERVER_DOWNLOAD_URL" ]]; then
```

Comment:
Personally I would find it confusing to need to specify `KOC_SERVER_DOWNLOAD_URL` when I always want to download the latest version.

This check maybe should be placed after trying to resolve the latest version.